packages/config/config/devices/0x015f/mh-p220_3.2-255.255

```diff
@@ -0,0 +1,280 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH-S220",
+	"description": "Micro Switch",
+	"devices": [
+		{
+			"productType": "0x2201",
+			"productId": "0x1252"
+		},
+		{
+			"productType": "0x2201",
+			"productId": "0x7202"
+		}
+	],
+	"firmwareVersion": {
+		"min": "3.2",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Switch 1",
+			"maxNodes": 5
+		},
+		"2": {
+			"label": "Switch 2",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
```

Comment:
Are you sure these didn't change? The params below seem to indicate those are now
1. Lifeline
2. Switch 1
3. Switch 2